bftengine/src/bftengine/DebugPersistentStorage.cpp

```diff
@@ -196,7 +196,7 @@ void DebugPersistentStorage::setDescriptorOfLastExecution(const DescriptorOfLast
   ConcordAssert(d.validRequests.numOfBits() <= maxNumOfRequestsInBatch);
 
   hasDescriptorOfLastExecution_ = true;
-  descriptorOfLastExecution_ = DescriptorOfLastExecution{d.executedSeqNum, d.validRequests};
+  descriptorOfLastExecution_ = DescriptorOfLastExecution{d.executedSeqNum, d.validRequests, d.timeInTicks};
```

Comment:
Generic comment for the persistent files, if timeservice is not enabled then everything should be either set to 0 or to some INVALID_TIME (which can be a time in far far future). So all these files should not assume that time service is available by default till 1.7 atleast.

Reply:
thanks, I'll make sure proper behavior in both cases